src/lib/viewers/image/ImageViewer.js

```diff
@@ -21,6 +21,10 @@ class ImageViewer extends ImageBaseViewer {
         if (this.isMobile) {
             this.handleOrientationChange = this.handleOrientationChange.bind(this);
         }
+
+        if (this.options.enableAnnotationsImageDiscoverability) {
```

Comment:
To make it clear that ImageViewer has it's own `annotationControlFSM`, perhaps we should always instantiate it's own instance and move the `this.options.enableAnnotationsImageDiscoverability` as a ternary for the initial state

Reply:
Yep good idea.